apps/ledger-live-mobile/src/newArch/features/Market/utils/index.ts

```diff
@@ -92,9 +98,13 @@ export const counterValueFormatter = ({
     return formattedNumber;
   }
 
-  // FIXME: HOW DID THIS WORK WHEN CURRENCY IS EMTPY
-  // PLEASE FIX
-  return formatter ? formatter.format(value) : value + "";
+  if (formatter) {
+    const formattedValue = formatter.format(value);
+    const upperCaseTicker = ticker?.trim()?.toLocaleUpperCase() || "";
+    return `${formattedValue} ${upperCaseTicker}`.trim();
+  }
+
+  return value + "";
```

Comment:
`value + "";` We can't justy do `return value ; `?

Reply:
counterValueFormatter is typed to return a string only I can update it to return a string or a number but I am not sure if it's what we want

Reply:
It might be better to keep only one type since it will mainly returns a string instead of just a number Lmk